addon.xml

```diff
@@ -1,17 +1,20 @@
 <?xml version="1.0" encoding="UTF-8" standalone="yes"?>
 <addon id="metadata.themoviedb.org.python"
        name="The Movie Database Python"
-       version="1.6.0"
+       version="1.6.1+matrix.1"
```

Comment:
Do not include the matrix.1 version tag nor change the xbmc.python version. These are both handled by the add-on submission process.